tools/prow-config-updater/pullrequest.go

```diff
@@ -0,0 +1,110 @@
+package main
```

Comment:
This can be in it's own package if you want something cleaner

Reply:
I think it's fine for now, will make improvement in following iterations...